### PR TITLE
Fix/sniff soc notifications intranet notification types

### PR DESCRIPTION
**Pull request by ilyeskallala**

### CODE REVIEW

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCNotificationsIntranetNotificationTypesTestIT.java [hunk 1]
```diff
@@ -0,0 +1,3490 @@
...
+import org.exoplatform.platform.qa.ui.core.context.BugInPLF;
+import org.exoplatform.platform.qa.ui.selenium.platform.*;
+import org.exoplatform.platform.qa.ui.selenium.platform.social.SpaceHomePage;
+import org.exoplatform.platform.qa.ui.selenium.platform.social.SpaceManagement;
+import org.exoplatform.platform.qa.ui.selenium.platform.social.SpaceSettingManagement;
+import org.exoplatform.platform.qa.ui.social.pageobject.*;
+
+@Tag("social")
+@Tag("sniff")
+@BugInPLF("SOC-5868")
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCNotificationsIntranetNotificationTypesTestIT.java]: This issue has been fixed. To be removed.

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCNotificationsIntranetNotificationTypesTestIT.java [hunk 2]
```diff
@@ -0,0 +1,3490 @@
...
+   * - The notification is displayed at the top of the list. Step Number: 3 Step
+   * Name: Step 3: check the message of the Like notification Step Description: -
+   * Check notification message in the notifications list and View All page Input
+   * Data: Expected Outcome: - The notification message is : LAST_AVATAR
+   * $USER_LIST like your activity $ACTIVITY $DATE Where : - $LAST_AVATAR is the
+   * thumbnail of User C - $USER_LIST is User B, User C - $ACTIVITY is the
+   * activity message/title - $DATE is the date of the last notification of User C
+   */
+  @Test
+  @BugInPLF("PLF-7656")
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCNotificationsIntranetNotificationTypesTestIT.java]: This issue has been fixed. To be removed.

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCNotificationsIntranetNotificationTypesTestIT.java [hunk 3]
```diff
@@ -0,0 +1,3490 @@
...
+   * notification message Step Description: - Click [View All] from the
+   * notifications list Input Data: Expected Outcome: - A Like notification is
+   * displayed in View All page - The activity message is : $LAST_AVATAR
+   * $LAST2_USERS and $COUNT more like your activity. $ACTIVITY $DATE Where : -
+   * $LAST_AVATAR is the thumbnail of User E - $LAST2_USERS is User E and User D -
+   * $COUNT is 2 - $ACTIVITY is the activity title/message - $DATE is the date of
+   * the activity.
+   */
+  @Test
+  @BugInPLF("PLF-7656")
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCNotificationsIntranetNotificationTypesTestIT.java]: This issue has been fixed. To be removed.

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCNotificationsIntranetNotificationTypesTestIT.java [hunk 4]
```diff
@@ -0,0 +1,3490 @@
...
+   * The notification is displayed at the top of the list Step Number: 3 Step
+   * Name: Step 3: Check comment notification in View All Step Description: -
+   * Check notification message in the list and in View All Input Data: Expected
+   * Outcome: - The notification message is : LAST_AVATAR $USER_LIST have
+   * commented on your activity $ACTIVITY $DATE Where : - $LAST_AVATAR is the
+   * thumbnail of User C - $USER_LIST is User B, User C - $ACTIVITY is the
+   * activity message/title - $DATE is the date of the last notification of User C
+   */
+  @Test
+  @BugInPLF("5852")
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCNotificationsIntranetNotificationTypesTestIT.java]: Wrong issue number.

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCNotificationsIntranetNotificationTypesTestIT.java [hunk 5]
```diff
@@ -0,0 +1,3490 @@
...
+   * - $DATE is the date of the notification Step Number: 3 Step Name: Step 3:
+   * Check View all Step Description: - Click [View All] from the bottom of the
+   * notification list - Check View All page Input Data: Expected Outcome: - The
+   * notification is displayed in the View All page Step Number: 4 Step Name: Step
+   * 4: Accept Step Description: - Click [Accept] Input Data: Expected Outcome: -
+   * The invitation is approved (User B is member of the space) - The
+   * notifications message is updated to : $AVATAR You joined $SPACE space. $DATE
+   */
+  @Test
+  public void test13_SpaceInvitationIntranet
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCNotificationsIntranetNotificationTypesTestIT.java]: Wrong issue number.

Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCNotificationsIntranetNotificationTypesTestIT.java]: Useless blank line.

Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCNotificationsIntranetNotificationTypesTestIT.java]: This issue has been fixed. To be removed.

Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCNotificationsIntranetNotificationTypesTestIT.java]: This issue has been fixed. To be removed.

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCNotificationsIntranetNotificationTypesTestIT.java (hunk 1 above)
Comment (ilyeskallala) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCNotificationsIntranetNotificationTypesTestIT.java]: done

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCNotificationsIntranetNotificationTypesTestIT.java (hunk 2 above)
Comment (ilyeskallala) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCNotificationsIntranetNotificationTypesTestIT.java]: done

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCNotificationsIntranetNotificationTypesTestIT.java (hunk 4 above)
Comment (ilyeskallala) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCNotificationsIntranetNotificationTypesTestIT.java]: tag removed

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCNotificationsIntranetNotificationTypesTestIT.java (hunk 5 above)
Comment (ilyeskallala) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCNotificationsIntranetNotificationTypesTestIT.java]: tag removed

**Pull request merged**